R/brm_package.R

```diff
@@ -9,15 +9,17 @@
 #'   it supports a simplified interfaced to reduce difficulty
 #'   and align with the best practices of the life sciences.
 #' @references
-#'   * Paul-Christian Bürkner (2017). brms: An R Package for Bayesian
+#'   * Bürkner PC (2017). brms: An R Package for Bayesian
 #'     Multilevel Models Using Stan.
-#'.    Journal of Statistical Software, 80(1), 1-28.
+#'.    J Stat Softw, 80(1), 1-28.
```

Comment:
Do these changes convert these citations to ASA style?

Reply:
Not yet. I try to consistently use abbreviations from the [NLM catalog](https://www.ncbi.nlm.nih.gov/nlmcatalog), since this is what is commonly used in journal publications. For the "Journal of Statistical Software" the abbreviation is "[J Stat Softw](https://www.ncbi.nlm.nih.gov/nlmcatalog/?term=J+Stat+Softw)", for "Therapeutic innovation & regulatory science" it is "[Ther Innov Regul Sci](https://www.ncbi.nlm.nih.gov/nlmcatalog/?term=Ther+Innov+Regul+Sci)". Here the reference is not (yet) coming from a BibTeX entry but I would also use these abbreviations in the .bib file. The change here just tries to correct inconsistencies in the way author names and journal names are treated in the two references. It's not really related to the new vignette. I suggest another PR to make referencing consistent in the package documentation (all refs coming from one central .bib file and formatted according to asa.csl.). This will be quite easy, there aren't many references so far, but there may be some more in the future, as in particular the vignettes may evolve.